src/explore-education-statistics-admin/src/pages/release/pre-release/components/PreReleaseInvitePlanModal.tsx

```diff
@@ -0,0 +1,120 @@
+import styles from '@admin/pages/release/pre-release/components/PreReleaseInvitePlanModal.module.scss';
+import ModalConfirm from '@common/components/ModalConfirm';
+import WarningMessage from '@common/components/WarningMessage';
+import React from 'react';
+import { PreReleaseInvitePlan } from '@admin/services/preReleaseUserService';
+
+interface Props {
+  invitePlan: PreReleaseInvitePlan;
+  isReleaseApproved: boolean;
+  onConfirm: () => void;
+  onCancel: () => void;
+  onExit: () => void;
+}
+
+const PreReleaseInvitePlanModel = ({
+  invitePlan,
+  isReleaseApproved,
+  onConfirm,
+  onCancel,
+  onExit,
+}: Props) => {
+  if (!invitePlan) {
+    return null;
+  }
+
+  return (
+    <ModalConfirm
+      title="Confirm pre-release invitations"
+      open
+      onConfirm={onConfirm}
+      onExit={onExit}
+      onCancel={onCancel}
+    >
+      <p>
+        Are you sure you want to invite these email addresses to access the
+        pre-release?
+      </p>
+
+      {invitePlan.invitable && invitePlan.invitable.length > 0 ? (
+        <>
+          <WarningMessage>
+            Email notifications will be sent{' '}
+            {isReleaseApproved
+              ? 'immediately.'
+              : 'when the release is approved for publication.'}
```

Comment:
This will split the text over multiple text nodes in the DOM, making it harder for tests to pick this up as a single line of text. Instead, we could use a template string like so:

```ts
{`Email notification will be sent ${isReleaseApproved 
  ? 'immediately.' 
  : 'when the release is approved for publication.'}`}
```

Reply:
Great tip, I was struggling with this in a test and was having difficulty preventing it getting reformatted over multiple lines. I've used your method of templating it now. Thanks.